db/recipe-sql.go

```diff
@@ -169,14 +169,16 @@ func (d *sqlRecipeDriver) SetState(id int64, state models.RecipeState) error {
 }
 
 func (d *sqlRecipeDriver) Find(filter *models.SearchFilter, page int64, count int64) (*[]models.RecipeCompact, int64, error) {
-	states := filter.States
```

Comment:
Need to update the frontend because the toolbar on the search results page still says "Active" instead of what should now be "All" when no states are selected.